src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -103,14 +103,9 @@ const PlansActionsDropdown: React.FunctionComponent<IPlansActionDropdownProps> =
         toggleOpen={toggleDeleteModal}
         mutateFn={() => deletePlanMutation.mutate(plan)}
         mutateResult={deletePlanMutation}
-        title="Delete migration plan"
+        title="Delete migration plan?"
         confirmButtonText="Delete"
-        body={
-          <>
-            Are you sure you want to delete the migration plan &quot;
-            <strong>{plan.metadata.name}</strong>&quot;?
-          </>
-        }
+        body={`Migration plan "${plan.metadata.name}" will be deleted.`}
         errorText="Error deleting plan"
```

Comment:
Does this string appear as an error message? If so, let's rephrase to align with PatternFly best practices:

Could not delete migration plan